lib/Runtime/Library/JavascriptSymbol.cpp

```diff
@@ -251,8 +257,7 @@ namespace Js
             *value = left->GetValue() == JavascriptSymbolObject::FromVar(right)->GetValue();
             break;
         default:
-            *value = JavascriptOperators::Equal_Full(right, left, requestContext);
-            break;
+            return FALSE;
```

Comment:
ah - good point. Fixed that.

---
In reply to: [99263244](https://github.com/Microsoft/ChakraCore/pull/2461#discussion_r99263244) [](ancestors = 99263244)